oaknational/Oak-Web-Application
# [CUR-561] Added cover page to docx generation

## Description
Added finalised version of cover page to docx file generation 

## Issue(s)

`CUR-561`

## How to test


1. Go to https://deploy-preview-2429--oak-web-application.netlify.thenational.academy/teachers/curriculum/docx-poc/select
2. Choose a subject via the autoComplete
3. Download the docx template - linked on the page
4. Download the rendered document
5. Check the cover page is present and correct

## Screenshots

<img width="429" alt="Screenshot 2024-05-17 at 11 52 08" src="https://github.com/oaknational/Oak-Web-Application/assets/235915/ff917446-6dd1-4193-8ee6-8ec05b5132d4">


## Checklist
